yarn.lock

```diff
@@ -4503,6 +4503,11 @@ electron-devtools-installer@^2.2.4:
     rimraf "^2.5.2"
     semver "^5.3.0"
 
+electron-process-type@^4.2.0:
```

Comment:
I'm not entirely sure if this is the way to go - I like the convenience though of having single function taking care of it - but library itself doesn't seem very popular / maintained (with only single contributor) & its for some reason heavier than this entire package :D Maybe we can extract that logic? 🤔

Reply:
This was my only qualm.

The only code we need to determine `renderer` versus `main` is:

```js
export const IS_MAIN_PROCESS = process.type === 'browser';
export const IS_RENDERER_PROCESS = process.type === 'renderer';
```

It mentions different results for sandboxing and other situations though, so I'm not sure my suggestion is as complete - but I think it's complete enough.

Reply:
I was hesitant to use this package, but I couldn't say with 100% certainty that `process.type` would always properly be set. I think we can just use process.type and remove the package and see from there.